lang/src/test/java/mumbler/truffle/parser/AnalyzerTest.java

```diff
@@ -19,8 +19,8 @@
     Namespace fileNs;
 
     @Before
-    public void setUp() throws Exception {
-        context = new MumblerContext();
+    public void setUp() {
+        context = new MumblerContext(null);
```

Comment:
This means setting with no MumblerLanguage yet, right? is that ok? Should there be a separate Constructor for that case explizitly?

Reply:
Yes, it's ok. MumblerContext is passed the language only when instantiated from the MumblerLanguage class (either by createContext, or parse). I will add a separate constructor for this.